src/coreclr/src/jit/lowerxarch.cpp

```diff
@@ -5594,6 +5594,7 @@ void Lowering::ContainCheckHWIntrinsic(GenTreeHWIntrinsic* node)
                             }
                         }
                     }
+                    break;
```

Comment:
can we use `unreached(); break;` in such cases?

Reply:
@sandreenko in the CI run, 15 coreclr tests have failed due to this added unreachable. So it seems it should be just a break here. I guess the fall through that was here before was benign, as the switch in the case below would end up in the default and nothing wrong would happen.